commcare_connect/static/js/project.js

```diff
@@ -63,3 +63,36 @@ function addAccuracyCircles(map, visit_data) {
 }
 
 window.addAccuracyCircles = addAccuracyCircles;
+
+function addCatchmentAreas(map, catchments) {
+  map.on('load', () => {
+    dummyCatchments.forEach((catchment) => {
```

Comment:
Is this able to be a single layer instead of one for each catchment area, like the forms above where they are one layer for all of them. Then we also dont have to do the dynamic naming

Reply:
Where does `dummyCatchments` come from?

Reply:
I forgot to rename the variable as i just added dummy content to test it. Will rename it.